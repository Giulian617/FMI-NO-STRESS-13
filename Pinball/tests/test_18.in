100 14536
11 31 7
5 11 3
89 11 7
48 31 10
32 5 1
29 48 8
71 48 10
51 48 1
74 5 10
75 74 5
45 89 7
42 5 3
88 89 1
57 74 3
53 89 9
27 51 8
97 45 7
34 88 6
8 31 2
50 57 10
22 74 9
23 11 9
44 74 4
62 29 5
33 62 6
17 75 9
10 23 5
73 62 8
82 75 5
1 11 10
4 33 4
9 88 2
58 88 9
13 22 6
30 75 1
2 4 7
78 34 10
52 78 7
56 33 3
67 23 2
87 27 8
24 30 8
72 9 2
99 10 5
6 72 10
92 57 4
84 10 1
81 62 1
80 5 10
7 89 1
35 23 10
19 87 5
64 72 6
94 33 8
60 44 6
90 2 5
40 87 5
95 2 5
12 40 3
39 58 5
77 88 7
91 30 8
36 94 10
54 74 2
21 73 2
18 17 4
61 51 7
55 5 9
46 19 9
96 82 10
41 46 6
70 67 10
43 48 6
93 95 8
63 32 1
85 77 3
66 27 10
83 24 10
49 52 9
68 97 5
38 7 7
20 73 3
86 54 5
37 11 1
69 1 8
65 61 7
59 8 9
25 72 7
100 38 7
47 67 1
14 9 10
26 14 4
28 53 9
16 13 6
98 34 1
15 36 8
79 67 1
3 40 2
76 36 8
