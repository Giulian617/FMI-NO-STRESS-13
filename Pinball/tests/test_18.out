21180051
382469508
799936897
127778529
375027354
1850725
868951735
858311518
138323027
538852914
607400681
691889571
815512307
355643255
789162892
593306336
545254289
742682239
855965183
525249692
455148430
366016699
461467623
64382589
257144525
778233927
497709990
928948423
705433171
765160177
668355244
14236707
127778529
742682239
192285834
846160535
702492823
910895302
418470405
338299942
985444645
702492823
824271796
985444645
920626160
267974149
571396603
866297552
